src/test/scala/edu/ie3/test/common/GridSupport.scala

```diff
@@ -260,12 +260,12 @@ trait GridSupport {
     // include at least a single node for voltage level determination
     val dummyNodeA = new NodeInput(
       UUID.randomUUID(),
-      s"Dummy nodeA in $subgridNo",
+      s"Dummy nodeA in $subgridNo + 1",
```

Comment:
Maybe you wanted to do this?

```suggestion
      s"Dummy nodeA in ${subgridNo + 1}",
```